dm/dm.go

```diff
@@ -71,13 +72,15 @@ var usage = func() {
 	fmt.Fprintln(os.Stderr, "Flags:")
 	flag.PrintDefaults()
 	fmt.Fprintln(os.Stderr)
-	os.Exit(1)
+	fmt.Fprintln(os.Stderr, "--stdin requires a tar archive of input files and a top level file name.")
```

Comment:
This seems odd to require a tar as an input format instead of straight text. This means that to run say the guestbook example for a container (which I think is the driving use case) you'd need to tar the file up and then pass it in? Or did I misunderstand?